test/e2e/set/scyllacluster/scyllacluster_replace.go

```diff
@@ -72,13 +73,32 @@ var _ = g.Describe("ScyllaCluster", func() {
 		ctx, cancel := context.WithTimeout(context.Background(), testTimeout)
 		defer cancel()
 
-		sc := scyllafixture.BasicScyllaCluster.ReadOrFail()
+		sc := f.GetDefaultScyllaCluster()
 		sc.Spec.Repository = e.scyllaImageRepository
 		sc.Spec.Version = e.scyllaVersion
 		sc.Spec.Datacenter.Racks[0].Members = 3
 
+		supportsExposing, err := scyllafeatures.Supports(sc, scyllafeatures.ExposingScyllaClusterViaServiceOtherThanClusterIP)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		if !supportsExposing {
```

Comment:
why here and not when generating the default cluster? ExposingScyllaClusterViaServiceOtherThanClusterIP seems to be a generic issue with configuration

Reply:
or should this be `ReplacingNodeUsingHostID`?

Reply:
I think there should only be one default, uninfluenced by the needs of specific tests. In other words if a test has some specific changes it should apply them directly like it does. 
Exposing requires `ReplacingNodeUsingHostID` so it's the same really, and I think since this influences the way we configure exposing in the test, the name is more fitting.

Reply:
> if I run a suite with --scyllacluster-nodes-broadcast-address-type=PodIP and a accidentally a wrong scylla version I don't want it to silently pass - it needs to fail or we can have a green CI job and wrongly think it's working

Agreed. I added a conditional label filter to the test runner instead, ptal.